Readme instructions for getting started are not accurate (npm run create Missing)

## 🐛 Bug: README References Non-Existent `npm run create/push/open` Scripts

### **Description**
The `README.md` instructs users to initialize and deploy the project using:

    npm run create
    npm run push
    npm run open

However, these scripts are **not defined** in the `package.json` file on the `main` branch. Running `npm run create` produces an error:

    npm ERR! Missing script: "create"

---

### **Steps to Reproduce**

1. Clone the repo:
       git clone git@github.com:ghelleks/email-agent.git
       cd email-agent

2. Install dependencies and follow the README:
       npm install
       npm run create

3. Observe the result:
       npm ERR! Missing script: "create"

---

### **Expected Behavior**
The commands listed in the README should correspond to actual npm scripts defined in `package.json`, or the documentation should be updated to reference the correct `clasp` commands.

---

### **Actual Behavior**
`npm run create`, `npm run push`, and `npm run open` are not defined. Existing npm scripts instead support multi-account setup and deployment:

    npm run setup:account
    npm run deploy:personal
    npm run open:personal
    npm run url:personal

---

### **Suggested Fix**

Option A – Documentation Update
Replace Step 3 in the README with direct `clasp` commands:

    clasp create --title "Gmail Labeler" --type standalone
    clasp push
    clasp open

Option B – Add npm Script Wrappers
Add the following entries to `package.json` for convenience:

    {
      "scripts": {
        "create": "clasp create --title 'Gmail Labeler' --type standalone",
        "push": "clasp push",
        "open": "clasp open"
      }
    }

---

### **Environment**
- OS: macOS (Apple Silicon)
- Node.js: v20+
- npm: v10+
- Branch: `main` (latest as of 2025-10-08)

---

### **Additional Context**
This appears to be a documentation drift — earlier revisions likely used single-account clasp scripts (`create/push/open`), while the current implementation focuses on multi-account deployment workflows (`personal`/`work`).

---

Labels: `bug`, `documentation`, `help wanted`
Title: README references outdated npm scripts (`create/push/open`) not defined in package.json

--- a/README.md
+++ b/README.md
@@ -33,6 +33,11 @@ clasp login --no-localhost
 
 This connects the command-line tool to your Google account.
 
+**Important**: After logging in, you must enable the Apps Script API:
+1. Visit https://script.google.com/home/usersettings
+2. Turn on "Google Apps Script API"
+3. Wait a few minutes for the change to propagate
+
 ### Step 2: Get a Gemini API Key
 
 1. Go to [Google Cloud Console](https://console.cloud.google.com/)
@@ -49,14 +54,14 @@ This connects the command-line tool to your Google account.
 git clone https://github.com/yourusername/email-agent.git
 cd email-agent
 
-# Create the Apps Script project
-npm run create
+# Create the Apps Script project (with source files in src/ directory)
+clasp create --title "Gmail Labeler" --type standalone --rootDir ./src
 
 # Upload code to Google's cloud
-npm run push
+clasp push
 
 # Open the Apps Script editor
-npm run open
+clasp open-script
 ```
 
 ### Step 4: Configure Settings
@@ -75,15 +80,16 @@ In the Apps Script editor that just opened:
 
 ### Step 5: Test It
 
-1. In the Apps Script editor, select **`run`** from the function dropdown
-2. Click the **"Run"** button (▶️)
-3. **Authorize the script** when prompted:
+1. In the Apps Script editor, click **`Main.gs`** in the Files list on the left
+2. Select **`run`** from the function dropdown at the top
+3. Click the **"Run"** button (▶️)
+4. **Authorize the script** when prompted:
    - Click "Review permissions"
    - Choose your Google account
    - Click "Advanced" → "Go to Gmail Labeler (unsafe)"
    - Click "Allow"
-4. Check the **"Execution log"** at the bottom for results
-5. Check your **Gmail** — you should see new labels applied to emails
+5. Check the **"Execution log"** at the bottom for results
+6. Check your **Gmail** — you should see new labels applied to emails
 
 **Testing tip**: Set `DRY_RUN=true` to analyze emails without actually applying labels.
 

